packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -759,8 +784,7 @@ export class CallContext {
   public deleteDeviceManagerUnparentedView(localVideoStream: LocalVideoStreamState): void {
     this.modifyState((draft: CallClientState) => {
       const foundIndex = draft.deviceManager.unparentedViews.findIndex(
-        (stream) =>
-          stream.source.id === localVideoStream.source.id && stream.mediaStreamType === localVideoStream.mediaStreamType
+        (stream) => stream.mediaStreamType === localVideoStream.mediaStreamType
```

Comment:
We don't match source ids because the source id changes when you switch cameras, right?